src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -312,106 +312,488 @@ void GetZoneEquipmentData(EnergyPlusData &state)
+            if (!state.dataHeatBal->Zone(zoneNum).IsControlled) {
+                ShowSevereError(state, format("{}{}=\"{}\"", RoutineName, CurrentModuleObject, thisZeqSplitter.Name));
+                ShowContinueError(
+                    state,
+                    format("..Zone Name={} is not a controlled zone. A ZoneHVAC:EquipmentConfiguration object is required for this zone.", zoneName));
+                state.dataZoneEquip->GetZoneEquipmentDataErrorsFound = true;
+                continue;
+            }
+
+            processZoneEquipSplitterInput(state, CurrentModuleObject, zeqSplitterNum, zoneNum, objectSchemaProps, objectFields, thisZeqSplitter);
+        }
+    } // end loop over zone equipment splitters
+
+    CurrentModuleObject = "SpaceHVAC:ZoneEquipmentMixer";
+    instances = ip->epJSON.find(CurrentModuleObject);
+    if (instances != ip->epJSON.end()) {
+        auto const &objectSchemaProps = ip->getObjectSchemaProps(state, CurrentModuleObject);
+        auto &instancesValue = instances.value();
+        int numZoneEqMixers = instancesValue.size();
+        state.dataZoneEquip->zoneEquipMixer.resize(numZoneEqMixers);
+        int zeqMixerNum = -1;
+        for (auto instance = instancesValue.begin(); instance != instancesValue.end(); ++instance) {
+            ++zeqMixerNum;
+            auto const &objectFields = instance.value();
+            auto &thisZeqMixer = state.dataZoneEquip->zoneEquipMixer[zeqMixerNum];
+            thisZeqMixer.Name = UtilityRoutines::makeUPPER(instance.key());
+            ip->markObjectAsUsed(CurrentModuleObject, instance.key());
+
+            std::string zoneName = ip->getAlphaFieldValue(objectFields, objectSchemaProps, "zone_name");
+            int zoneNum = UtilityRoutines::FindItemInList(zoneName, state.dataHeatBal->Zone);
+            if (zoneNum == 0) {
+                ShowSevereError(state, format("{}{}=\"{}\"", RoutineName, CurrentModuleObject, thisZeqMixer.Name));
+                ShowContinueError(state, format("..Zone Name={} not found, remaining items for this object not processed.", zoneName));
+                state.dataZoneEquip->GetZoneEquipmentDataErrorsFound = true;
+                continue;
+            }
+            if (!state.dataHeatBal->Zone(zoneNum).IsControlled) {
+                ShowSevereError(state, format("{}{}=\"{}\"", RoutineName, CurrentModuleObject, thisZeqMixer.Name));
+                ShowContinueError(
+                    state,
+                    format("..Zone Name={} is not a controlled zone. A ZoneHVAC:EquipmentConfiguration object is required for this zone.", zoneName));
+                state.dataZoneEquip->GetZoneEquipmentDataErrorsFound = true;
+                continue;
+            }
+
+            processZoneEquipMixerInput(state, CurrentModuleObject, zeqMixerNum, zoneNum, objectSchemaProps, objectFields, thisZeqMixer);
+        }
+    } // end loop over zone equipment splitters
+
+    CurrentModuleObject = "AirLoopHVAC:SupplyPath";
```

Comment:
This an following objects are unchanged, not sure why diff is showing as new (even with whitespace hidden).